OracleLinux/7/scripts/lamp.sh

```diff
@@ -0,0 +1 @@
+../../../LAMP/scripts/lamp.sh
```

Comment:
I would copy this script into the current location so we can remove the `LAMP` directory in the future.